jdi-light-mobile-tests/src/main/java/nativeapp/ios/contacts/ContactsApp.java

```diff
@@ -0,0 +1,7 @@
+package nativeapp.ios.contacts;
+
+public class ContactsApp {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ContactsApp' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5531163)